### DISCUSSION TITLE
[Help] How can I allow for both title and tag input?

### DISCUSSION THREAD

**Discussion by ThSGM** [zk-org/zk-nvim#168] [Help]
### Verify

- [X] I searched the existing discussions for help

### How can we help you?

I'd like to make a command to create a new file querying the title and the tags. I thought you could do this with this (see the definition below for <leader>zn) but it throws up an error (not an editor command). There is no documentation on this kind of request of two inputs. 

```
return {
  {
    "zk-org/zk-nvim",
    config = function()
      require("zk").setup({
        -- See Setup section below
      })
    end,
    keys = {
      {
        "<leader>zn",
        "<cmd>zknew { title = vim.fn.input('title: '), tags = vim.fn.input('tags (:tag:)'}<cr>",
        { noremap = true, silent = true },
        desc = "zk new"
      },
      {
        "<leader>zo",
        "<Cmd>ZkNotes { sort = { 'modified' } }<CR>",
        desc = "zk open"
      },
      {
        "<leader>zt",
        "<Cmd>ZkTags<CR>",
        desc = "zk open tags"
      },
      {
        "<leader>zf",
        "<Cmd>ZkNotes { sort = { 'modified' }, match = { vim.fn.input('Search: ') } }<CR>",
        desc = "zk search"
      },
    },
  },
}
```

**Comment by tjex** [comment]
Just looks to be a simple typo? :)

`<cmd>zknew` should be `<cmd>ZkNew`

**Reply by ThSGM** [reply]
No, I had tried that before as well and it didn't work:

<img width="754" alt="image" src="https://github.com/zk-org/zk-nvim/assets/3396578/d2b64379-0e07-4d4b-ae46-f7d803253dd8">

Can you check it works on your system?

**Reply by tjex** [reply]
hey. sorry for delayed reply. I was on holiday.

I already have a keymap that's effectlvely the same, [see here](https://github.com/tjex/dotfiles/blob/879abad39786fa09ab20d844687c6f40b1b92007/.config/nvim/after/plugin/zk.lua#L49). But I don't use lazyvim. Maybe it's the `{noremap...}` table. None of your other keymaps have this entry. So that's also a point of difference.

Also, as a side note, please always copy and paste code as text and format appropriately (code blocks with language definition).
I can't even see the end of the code line in question (line 13). Plus the font is wonky to my eyes. 

If you can't copy the error message, then that can be an image. 

thanks 👍
